src/components/ClassCard/ClassCard.stories.js

```diff
@@ -1,21 +1,20 @@
 import React from "react";
-
 import ClassCard from "./ClassCard";
 
 export default {
-  title: "Example/ClassCard",
+  title: "ClassCard",
   component: ClassCard,
-  argTypes: {
-    backgroundColor: { control: "color" },
-  },
 };
 
 const Template = (args) => <ClassCard {...args} />;
 
-export const Primary = Template.bind({});
-Primary.args = {
+export const Default = Template.bind({});
+Default.args = {
   id: 1,
   name: "Good morning Sunshine",
   duration: "00:30",
   intervalTime: "00:01",
+  onRemoveClassClick() {
+    console.log("Clicked");
+  },
```

Comment:
console.log 🧐

Reply:
This actually is called in storybook and displayed, that the button was clicked. Should stay ;-)